src/js/messaging/containers/Compose.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { messageCategories, messageCategoryError } from '../config.js';
+import MessageCategory from '../components/compose/MessageCategory';
+import { setCategory } from '../actions/compose.js';
+
+class Compose extends React.Component {
+  constructor() {
+    super();
+    this.dispatchCategoryChange = this.dispatchCategoryChange.bind(this);
+  }
+
+  dispatchCategoryChange(event) {
+    this.props.dispatch(setCategory(event));
```

Comment:
This is fine here, since we're only using one action creator here. But for future reference, the way to do this idiomatically seems to be passing into `connect` a `mapDispatchToProps` that makes use of `bindActionCreators`. In Rx, we passed an object of action creators for shorthand, and maybe when we need that again, we can do something like:

```
const actionCreators = {
  setCategory,
  // Any other action creators...
  ...
}

export default connect(mapStateToProps, actionCreators)(Compose);
```